src/components/Search/search-result.jsx

```diff
@@ -0,0 +1,106 @@
+import { Link } from 'gatsby';
+import { default as React } from 'react';
+import { Paragraph } from '@smallstep/step-ui';
+import { makeStyles } from '@material-ui/styles';
+import {
+  connectStateResults,
+  Highlight,
+  Hits,
+  Index,
+  Snippet,
+} from 'react-instantsearch-dom';
+import { Box, ListItem, ListItemText } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  resultsBorder: {
+    borderColor: '#D3D3D3',
+    marginBottom: -16,
+    marginTop: -2,
+  },
+  hits: {
+    marginLeft: -30,
+    marginRight: 10,
+    '& mark ': {
+      backgroundColor: '#84A8FF',
+    },
+    '& ul ': {
+      listStyle: 'none',
+    },
+    '& a:-webkit-any-link ': {
+      textDecoration: 'none',
+      color: 'inherit',
+    },
+  },
+});
+
+const HitCount = connectStateResults(({ searchResults }) => {
+  const hitCount = searchResults && searchResults.nbHits;
+  const theme = useStyles();
+  return hitCount > 0 ? (
+    <Box borderBottom={1} textAlign={'right'} className={theme.resultsBorder}>
+      <Box mr={2}>
+        {hitCount} result{hitCount !== 1 ? `s` : ``}
+      </Box>
+    </Box>
+  ) : (
+    <Box
+      className={theme.resultsBorder}
```

Comment:
I'm wondering where we add borders if the `Paper` component would be more appropriate. Will try to get it running and see!